static/scss/geosuggest.scss

```diff
@@ -0,0 +1,69 @@
+// https://github.com/ubilabs/react-geosuggest/blob/master/src/geosuggest.css
```

Comment:
I actually already tried importing the file from the library in `static/js/entry/style.js`, but then discovered that we needed to make some changes to the styling to make it resemble the Material UI input fields. I figured that copying the file into the repo and making changes there was the best solution, but we could also import the default styles and then override them as necessary. Do you have a preference?